package.json

```diff
@@ -8,5 +8,8 @@
   },
   "devDependencies": {
     "bower": "1.8.8"
+  },
+  "scripts": {
```

Comment:
Why is this script necessary?

Reply:
This is necessary to provide the `qunitjs` files, because the `test/encoding.html` includes them from the `node_modules` folder of the bower dependency `js-cookie`.